fiscal_epos_print/models/account_journal.py

```diff
@@ -0,0 +1,23 @@
+from odoo import fields, models
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    fiscalprinter_payment_type = fields.Selection(
+        [
+            ('0', 'Cash'),
+            ('1', 'Cheque'),
+            ('2', 'Credit/Credit Card'),
```

Comment:
Qui si intendono due metodi distinti (Accredito/Carta di credito) ?
Lo chiedo perché nella traduzione ho trovato solo "Carta di credito".

Reply:
Ah, non lo so. @robyf70 @GSLabIt ?

Reply:
il tipo pagamento 2 può avere diversi indici di pagamento (index):
0 - credito al cliente (non incassato)
1 - transazione elettronica (bancomat / carta di credito)

Da qui l'indicazione `Credit / Credit Card`. Si potrebbe anche modificare in `Credit / Electronic payment` o similare

Reply:
Quindi comunque la traduzione corretta sarebbe `Credito/Carta di credito`. Modificato

Reply:
@GSLabIt @eLBati 
Ah, ok.
Per quanto riguarda l'indice 0 terrei però un più generico "Non riscosso", usato nelle specifiche AdE (e anche [qui](https://wikitswellness.teamsystem.com/mediawiki/index.php/RT) ad es.)
Per lo stesso motivo terrei generico anche l'indice 1, si potrebbe quindi modificare con:
`Not collected / Electronic payment`

Reply:
Modificato!